lumigator/python/mzai/backend/.alembic/env.py

```diff
@@ -0,0 +1,89 @@
+import os
+from logging.config import fileConfig
+
+from alembic import context
+from backend.records.base import BaseRecord
+from backend.records.datasets import *
+from backend.records.jobs import *
+from sqlalchemy import engine_from_config, pool
+
+"""
+NOTE: Do NOT remove imports for the data models: backend.records.{package}
+Some may appear to be unused, but they are required in order for the BaseRecord's
+metadata to be populated correctly when it is used by Alembic as 'target_metadata'.
+
+If a new data model package is added, then it MUST be manually imported here!
+"""
+
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
```

Comment:
is alembic logging separate from the app-level logging we do via logguru? would we want it to be the same?

Reply:
The Alembic 'stuff' is totally separate from the app running, so I think it's probably fine? 🤔 A lot of the contents of that file is scaffolding from the `alembic init` command. In the `env.py` the main 'changes' are the imports at the top (with the note about them having to be there), and:

```
target_metadata = BaseRecord.metadata

# Override the SQLAlchemy URL with the one we have stored in our environment.
config.set_main_option(
    'sqlalchemy.url',
    os.environ.get("SQLALCHEMY_DATABASE_URL", config.get_main_option("sqlalchemy.url")))
```

it's a manual step you'd run if you made schema changes to the data models, to generate the migration version, so any logging would be something you'd probably want to see in your terminal window.

Reply:
Awesome, thanks for the explanation!